src/alcotest/alcotest.mli

```diff
@@ -111,6 +111,9 @@ val reject : 'a testable
 val check : 'a testable -> string -> 'a -> 'a -> unit
 (** Check that two values are equal. *)
 
+val check_l : t:'a testable -> msg:string -> expected:'a -> actual:'a -> unit
```

Comment:
```suggestion
val check_l : 'a testable -> msg:string -> expected:'a -> actual:'a -> unit
```
I'm not sure this first label is adding much in the way of code-descriptiveness – `t` values in other label-heavy libraries like Base and Core are generally not labelled. 

Perhaps we should drop it? It will still be possible to pass all four arguments in any order.